history-of-quakers/modernized/26-chapter-24.adoc

```diff
@@ -61,13 +75,19 @@ and charity, to the glory of God, and the benefit of these nations.
 May the King of kings make your reign long and glorious,
 to which temporal blessing we shall pray for your eternal happiness.
 
+[.signed-section-context-close]
 Signed on behalf and by appointment of the aforesaid people, at a meeting in London,
 the 10th of the second month, 1702.
 
+--
+
 This address was very favorably received by the queen, and after it was read to her,
 she answered,
 
+[quote]
```

Comment:
These conversations I really struggled with, whether to keep them just in quotation marks or to use a block quote. They were offset in the orginal PDF, so that's what made me think I should use a block quote. IDK!!! 😄